cmd/litcli/ln.go

```diff
@@ -659,8 +659,14 @@ var addInvoiceCommand = cli.Command{
 				"cannot be used at the same time as --asset_id",
 		},
 		cli.Uint64Flag{
-			Name:  "asset_amount",
-			Usage: "the amount of assets to receive",
+			Name: "asset_amount",
+			Usage: "the amount of assets to receive, mutually " +
+				"exclusive to msat_amount",
+		},
+		cli.Uint64Flag{
+			Name: "msat_amount",
```

Comment:
If you look at `litcli ln addinvoice`, you see that we already have all flags from `commands.AddInvoiceCommand.Flags`:

```
   --amt value                      the amt of satoshis in this invoice (default: 0)
   --amt_msat value                 the amt of millisatoshis in this invoice (default: 0)
```

So we should use those.